File: Algo/Common/MissingNumber/missingNumber.py
def missingNumber(arr):
    n = len(arr) + 1
    sumOfarray = sum(arr)
    sumOf_N_numbers = (n * (n + 1))//2
    missingOne = sumOf_N_numbers - sumOfarray
    return missingOne

arr = [1,2,3,5,6]
print(missingNumber(arr))

File: Algo/Common/MissingNumber/MissingNumber.java
class MissingNumber{  

    public static void main(String args[]){ 
        int[] arr = new int[] {1,3,4,5,6};
        System.out.println(missingNumber(arr));
    }  

    public static int missingNumber(int[] arr) {
        int arraySum = 0;
        int expectedSum = 0;
        for (int i = 1; i <= arr.length; i++) {
            expectedSum += i;
            arraySum += arr[i - 1];
        }
        return expectedSum - arraySum;
    } 
} 
